content/en/blog/_posts/2024-12-11-Kubernetes-v1-32-Release.md

```diff
@@ -0,0 +1,302 @@
+
+### Quality of life improvements on nodes and sidecar containers update
+
+Some highlights beyond KEPs to take a look at, by [SIG Node](https://github.com/kubernetes/community/tree/master/sig-node).
+
+1. The systemd watchdog capability is now utilized to restart the kubelet when its health check fails, while limiting the maximum number of restarts within a given time period. This enhances the reliability of the kubelet to some extent. You can find more details in the pull request [#127566](https://github.com/kubernetes/kubernetes/pull/127566).
+
+2. In cases when an image pull back-off error is encountered, the message displayed in the pod status has been improved to be more human-friendly and to indicate details about why the pod is in this condition.
+The image pull error is now appended to the pods `status.containerStatuses[*].state.waiting.message` when in image pull back-off (`reason` is `ImagePullBackOff`). This change provides more context to the user, making it easier to identify the root cause of the issue. You can find more details in the pull request [#127918](https://github.com/kubernetes/kubernetes/pull/127918).
+
+3. The Sidecar containers feature is targeting graduation to Stable in 1.33 as the [remaining work items](https://github.com/kubernetes/enhancements/issues/753#issuecomment-2350136594) are being tracked along with feedback from users.
+
+## Highlights of features graduating to Stable
+
+_This is a selection of some of the improvements that are now stable following the v1.32 release._
+
+### Custom Resource field selectors
+
+Custom resource field selector allows developers to add field selectors to custom resources, mirroring the functionality available for built-in Kubernetes objects. This allows for more efficient and precise filtering of custom resources, promoting better API design practices.
+
+This work was done as a part of [KEP #4358](https://github.com/kubernetes/enhancements/issues/4358), by [SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery).
+
+### Support to size memory backed volumes
+
+The feature makes it possible to dynamically size memory-backed volumes based on pod resource limits. This improves the workload's portability and overall node resource utilization for the users.
+    
+This work was done as a part of [KEP #1967](https://github.com/kubernetes/enhancements/issues/1967), by [SIG Node](https://github.com/kubernetes/community/tree/master/sig-node).
+
+### Job API managed-by mechanism
+
+Support of the `managedBy` field has been added for Jobs and it is stable in 1.32 release. This feature enables external controllers (like [Kueue](https://kueue.sigs.k8s.io/)) to manage Job synchronization, offering greater flexibility and integration with advanced workload management systems.
```

Comment:
```suggestion
Support of the `managedBy` field has been added for Jobs and it was promoted to beta and on by default in 1.32 release. This feature enables external controllers (like [Kueue](https://kueue.sigs.k8s.io/)) to manage Job synchronization, offering greater flexibility and integration with advanced workload management systems.
```

Reply:
We should probably also move this below to the beta list.

Reply:
Thank you, it will be fixed on the upcoming commit!